modules/ui/source_subfield.js

```diff
@@ -0,0 +1,170 @@
+                        _sourceValue = undefined;
+
+                        let t = {};
+                        t[sourceKey] = undefined;
+                        dispatch.call('change', this, t);
+
+                        renderSourceInput(selection);
+                    })
+                    .call(svgIcon('#iD-operation-delete'));
+
+                wrap
+                    .append('input')
+                    .attr('type', 'text')
+                    .attr('class', 'field-source-value')
+                    .on('blur', changeSourceValue)
+                    .on('change', changeSourceValue);
+            });
+
+        entriesEnter
+            .style('margin-top', '0px')
+            .style('max-height', '0px')
+            .style('opacity', '0')
+            .transition()
+            .duration(200)
+            .style('margin-top', '10px')
+            .style('max-height', '240px')
+            .style('opacity', '1')
+            .on('end', function() {
+                d3_select(this)
+                    .style('max-height', '')
+                    .style('overflow', 'visible');
+            });
+
+        entries = entries.merge(entriesEnter);
+
+        entries.order();
+
+        // allow removing the entry UIs even if there isn't a tag to remove
+        entries.classed('present', true);
+
+        utilGetSetValue(entries.select('.field-source-value'), function(d) {
+                return typeof d === 'string' ? d : '';
+            })
+            .attr('title', function(d) {
+                return Array.isArray(d) ? d.filter(Boolean).join('\n') : null;
+            })
+            .attr('placeholder', function(d) {
+                return Array.isArray(d) ? t('inspector.multiple_values') : t('inspector.field_source_placeholder');
+            })
+            .classed('mixed', function(d) {
+                return Array.isArray(d);
+            });
+    }
+
+    function changeSourceValue(d3_event, d) {
+        let value = context.cleanTagValue(utilGetSetValue(d3_select(this))) || undefined;
+        // don't override multiple values with blank string
+        if (!value && Array.isArray(d.value)) return;
+
+        let t = {};
+        t[sourceKey] = value;
+        d.value = value;
+        dispatch.call('change', this, t);
+    }
+
+    function addSource(d3_event, d) {
+        d3_event.preventDefault();
+        
+        if (typeof _sourceValue !== 'string' && !Array.isArray(_sourceValue)) {
+            _sourceValue = '';
+        }
+        sourceInput.call(renderSourceInput);
+        
+    }
+
+    sourceSubfield.button = function(labelEnter, container) {
+        let sourceButtonTip = uiTooltip()
+                .title(() => t.append('inspector.field_source'))
+                .placement('left');
+    
+        labelEnter
+        .append('button')
+        .attr('class', 'source-icon')
+        .attr('title', 'source-button')
+        .call(sourceButtonTip)
+        .call(svgIcon('#fas-book'));
```

Comment:
The Help button on the right side of the map uses [an icon that also looks like a book](https://github.com/OpenHistoricalMap/iD/blob/a5a46ae34e3eef0ba5b69e7f89d7834fc3c5073d/svg/iD-sprite/icons/icon-help.svg). This one doesn’t have a question mark on it, but some users might still confuse it with the ℹ️ help button directly to the right. We don’t have to come up with the ideal icon yet, but let’s keep looking for possible alternatives.

Reply:
I also thought about these other icons, do you think any of them is better than the book? I think the "@" one is the best from these ones.

<img src="https://github.com/OpenHistoricalMap/iD/assets/62030098/7a5777e5-b4bd-4abb-850e-9189b03ad4c8" width="20px" height="20px">
<img src="https://github.com/OpenHistoricalMap/iD/assets/62030098/b7b30b38-ab31-4b8b-a240-9c5bcfdf6e08" width="20px" height="20px">
<img src="https://github.com/OpenHistoricalMap/iD/assets/62030098/059957c9-0ab9-435c-ae1d-a197c6970086" width="20px" height="20px">
<img src="https://github.com/OpenHistoricalMap/iD/assets/62030098/a0c6e88b-cdf0-4ac6-9d9a-67f188607d25" width="20px" height="20px">

Reply:
Yeah, that @ one seems to be different enough from other icons we use. It’s sort of like @-mentioning someone, in this case, a source. You could take a look around at other sites for inspiration: academic sites, reference sites, research databases, etc. What symbols or objects do they use to represent sources or citations, if anything?